job-service-scheduled-executor/src/main/java/com/hpe/caf/services/job/scheduled/executor/ScheduledExecutorConfig.java

```diff
@@ -64,6 +67,11 @@ public static String getTrackingPipe() {
         return getPropertyOrEnvVar("CAF_TRACKING_PIPE");
     }
 
+    public static String useNewQueueMessageFormat() {
+        final String outPutFormatVersion = getPropertyOrEnvVar("JOB_SERVICE_MESSAGE_OUTPUT_FORMAT").toUpperCase(Locale.ROOT);
```

Comment:
Updated with String instead of boolean with ["V3", "V4"] as accepted value and V3 as the default one.